stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: nginx-ingress
-version: 1.1.5
-appVersion: 0.21.0
+version: 1.2.0
+appVersion: 0.22.0
```

Comment:
please update the readme too for consistency.

Reply:
Updated readme default